tests/unit/lxd/test_lxc.py

```diff
@@ -2724,7 +2744,7 @@ def test_is_pro_enabled_process_error(fake_process):
     assert len(fake_process.calls) == 1
 
 
-def test_attach_pro_subscription_success(fake_process):
+def test_attach_pro_subscription_success(mock_pro_config_handling, fake_process):
```

Comment:
```suggestion
@pytest.mark.usefixtures("mock_pro_config_handling")
def test_attach_pro_subscription_success(fake_process):
```
nit: for tests that don't actually use the value of the fixture, you can request them from a fixture instead